### PR TITLE
Test with Blacklight 6

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: lib/blacklight/access_controls/enforcement.rb [hunk 1]
```diff
@@ -23,6 +23,14 @@ def current_ability
...
       end
 
+      def except(*)
+        super.tap { |new_builder| new_builder.current_ability = current_ability }
+      end
+
+      def append(*)
+        super.tap { |new_builder| new_builder.current_ability = current_ability }
+      end
+
```
Comment (awead) [inline lib/blacklight/access_controls/enforcement.rb]: These two methods do the exact same thing. Can you document their differences?

Comment (jcoyne) [inline lib/blacklight/access_controls/enforcement.rb]: No, they don't do the same thing. One calls `append` (`super`) and the other calls `except`

Comment (awead) [inline lib/blacklight/access_controls/enforcement.rb]: @jcoyne I don't understand. Lines 27 and 31 are the same.

Comment (jcoyne) [inline lib/blacklight/access_controls/enforcement.rb]: @awead the `super` keyword calls the current method in the super class. On line 27 that would be `append` (https://github.com/projectblacklight/blacklight/blob/master/lib/blacklight/search_builder.rb#L54). On line 31 it calls `except` (https://github.com/projectblacklight/blacklight/blob/master/lib/blacklight/search_builder.rb#L76)

Comment (awead) [inline lib/blacklight/access_controls/enforcement.rb]: Oh, I see. thanks.

**Pull request merged**